libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/StreamingExtensions/BotFrameworkStreamingExtensionsAdapter.cs

```diff
@@ -0,0 +1,321 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Protocol;
+using Microsoft.Bot.Protocol.WebSockets;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.WebApi.StreamingExtensions
```

Comment:
Before delving too far into WebAPI, might I suggest we get to closure on the .Net Core implementation first? I suspect nearly all of my questions/comments there apply here as well.